packages/app/src/systems/Bridge/pages/BridgeHome.tsx

```diff
@@ -42,7 +42,7 @@ export const BridgeHome = ({ children }: BridgeHomeProps) => {
 
 const styles = {
   content: cssObj({
-    maxWidth: '$sm',
+    maxWidth: '$xl',
```

Comment:
should be `$sm`

Reply:
if it is small then the width of the transaction list items can not match the width on figma

Reply:
the width on figma is the same in both Bridge and Transactions page